ui/apps/platform/src/Containers/Administration/Events/AdministrationEventsTable.tsx

```diff
@@ -56,27 +57,36 @@ function AdministrationEventsTable({ events }: AdministrationEventsTableProps):
                             }}
                         >
                             <Tr>
-                                <Td dataLabel="Level icon">{getLevelIcon(level)}</Td>
+                                <Td dataLabel="Domain" modifier="nowrap">
+                                    <Link to={`/main/administration-events/${id}`}>{domain}</Link>
+                                </Td>
+                                <Td dataLabel="Resource type" modifier="nowrap">
+                                    {resourceType}
+                                </Td>
                                 <Td dataLabel="Level">
-                                    <Link to={`/main/administration-events/${id}`}>
-                                        {getLevelText(level)}
-                                    </Link>
+                                    <IconText
+                                        icon={getLevelIcon(level)}
+                                        text={getLevelText(level)}
+                                    />
                                 </Td>
-                                <Td dataLabel="Domain">{domain}</Td>
-                                <Td dataLabel="Resource type">{resourceType}</Td>
                                 <Td dataLabel="Event last occurred at" modifier="nowrap">
                                     {lastOccurredAt}
                                 </Td>
-                                <Td dataLabel="Occurrences" className="pf-u-text-align-right">
+                                <Td dataLabel="Count" className="pf-u-text-align-right">
                                     {numOccurrences}
                                 </Td>
                             </Tr>
                             <Tr>
-                                <Td />
-                                <Td colSpan={6}>
+                                <Td colSpan={5}>
                                     <ExpandableRowContent>
                                         <Flex direction={{ default: 'column' }}>
-                                            {hint && <p>{hint}</p>}
+                                            {hint && (
+                                                <div>
+                                                    {hint.split('\n').map((line) => (
+                                                        <p key={line}>{line}</p>
+                                                    ))}
```

Comment:
Interesting that the API assumes CLI output in its response.

Reply:
Yeah, I created an offline test file to make sure that `split` method always returns an array of strings.